docs/install_guides/installing-openvino-images.md

```diff
@@ -4,9 +4,6 @@ You may install Intel® Distribution of OpenVINO™ toolkit from images and repo
 
 * [Docker](installing-openvino-docker-linux.md)
 * [Docker with DL Workbench](@ref workbench_docs_Workbench_DG_Run_Locally)
-* [APT](installing-openvino-apt.md)
-* [YUM](installing-openvino-yum.md)
-* [Anaconda Cloud](installing-openvino-conda.md)
 * [Yocto](installing-openvino-yocto.md)
 * [PyPI](installing-openvino-pip.md)
```

Comment:
This file is not used anymore. Let's remove it from the directory and add an item in the change log.

Reply:
@msmykx-intel

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/12222/commits/e8b5e2930ce3aac70a26a947a52847198392fb50

Info added to PR description